src/nfts/NftsInfoCarousel.test.tsx

```diff
@@ -133,40 +131,4 @@ describe('NftsInfoCarousel', () => {
 
     expect(queryByTestId('ViewOnExplorer')).toBeNull()
   })
-
-  it('getGatewayUrl', () => {
-    expect(getGatewayUrl(mockNftAllFields)).toEqual(mockNftAllFields.media[0].gateway)
-  })
-
-  it('onImageLoad success', () => {
-    onImageLoad(mockNftAllFields, NftOrigin.TransactionFeed, false)
-    expect(ValoraAnalytics.track).toBeCalledWith('nft_image_load', {
-      tokenId: mockNftAllFields.tokenId,
-      contractAddress: mockNftAllFields.contractAddress,
-      url: mockNftAllFields.media[0].gateway,
-      origin: NftOrigin.TransactionFeed,
-      error: false,
-    })
-    expect(ValoraAnalytics.track).toBeCalledTimes(1)
-    expect(Logger.info).toBeCalledWith(
-      NftOrigin.TransactionFeed,
-      `ContractAddress=${mockNftAllFields.contractAddress}, TokenId: ${mockNftAllFields.tokenId}, Loaded image from ${mockNftAllFields.media[0].gateway}`
-    )
-  })
-
-  it('onImageLoad error', () => {
-    onImageLoad(mockNftAllFields, NftOrigin.NftsInfoCarouselMain, true)
-    expect(ValoraAnalytics.track).toBeCalledWith('nft_image_load', {
-      tokenId: mockNftAllFields.tokenId,
-      contractAddress: mockNftAllFields.contractAddress,
-      url: mockNftAllFields.media[0].gateway,
-      origin: NftOrigin.NftsInfoCarouselMain,
-      error: true,
-    })
-    expect(ValoraAnalytics.track).toBeCalledTimes(1)
-    expect(Logger.error).toBeCalledWith(
-      NftOrigin.NftsInfoCarouselMain,
-      `ContractAddress=${mockNftAllFields.contractAddress}, TokenId: ${mockNftAllFields.tokenId}, Failed to load image from ${mockNftAllFields.media[0].gateway}`
-    )
-  })
```

Comment:
i removed these tests because i removed these abstractions (we were unhappy with them anyway)

i decided not to add tests for these. what we actually want to test, is when FastImage loads or fails to load the image, that the correct analytics is dispatched. i'd argue that that is not what these tests were doing anyway, they are testing a function but not testing when the function is triggered. i considered mocking out FastImage to get it to throw errors etc but it seems too much like testing the internals of FastImage